tests/Traits/FakeCacheImplementationTrait.php

```diff
@@ -3,51 +3,13 @@
 namespace Unleash\Client\Tests\Traits;
 
 use Psr\SimpleCache\CacheInterface;
+use Symfony\Component\Cache\Adapter\NullAdapter;
+use Symfony\Component\Cache\Psr16Cache;
 
 trait FakeCacheImplementationTrait
 {
     private function getCache(): CacheInterface
     {
-        return new class implements CacheInterface {
-            public function get($key, $default = null)
-            {
-                return $default;
-            }
-
-            public function set($key, $value, $ttl = null): bool
-            {
-                return true;
-            }
-
-            public function delete($key): bool
-            {
-                return true;
-            }
-
-            public function clear(): bool
-            {
-                return true;
-            }
-
-            public function getMultiple($keys, $default = null)
-            {
-                return $default;
-            }
-
-            public function setMultiple($values, $ttl = null): bool
-            {
-                return true;
-            }
-
-            public function deleteMultiple($keys): bool
-            {
-                return true;
-            }
-
-            public function has($key): bool
-            {
-                return false;
-            }
-        };
+        return new Psr16Cache(new NullAdapter());
```

Comment:
`symfony/cache` is already required as a dev dependency, so we can use it's `NullAdapter` which does exactly the same thing, and it gracefully supports `psr/simple-cache` the varied types used in 1.x and 3.x!